src/javaRosa.js

```diff
@@ -824,16 +824,17 @@ define([
         options.id = id;
 
         var widget = widgets.base(mug, options);
-        var $input = $("<textarea></textarea>")
+        var $input = $("<div contenteditable>")
```

Comment:
I wonder how much jQuery manipulation we do with these inputs. What do you think of making a wrapper object for the div that gives it the same interface as a regular input? Then you wouldn't have to go around an change every piece of code that does `$input.val()`, for example.